backend/dataall/modules/mlstudio/services/services.py

```diff
@@ -0,0 +1,207 @@
+            response = SagemakerStudioClient.get_sagemaker_studio_domain(
+                AwsAccountId=env.AwsAccountId,
+                region=env.region
+            )
+            existing_domain = response.get('DomainId', False)
+
+            if not existing_domain:
+                raise exceptions.AWSResourceNotAvailable(
+                    action='Sagemaker Studio domain',
+                    message='Add a VPC to your environment and update the environment stack '
+                            'or create a Sagemaker studio domain on your AWS account.',
+                )
+
+            sagemaker_studio_user = SagemakerStudioUser(
+                label=request.label,
+                environmentUri=env.environmentUri,
+                description=request.description,
+                sagemakerStudioUserProfileName=slugify(request.label, separator=''),
+                sagemakerStudioUserProfileStatus='PENDING',
+                sagemakerStudioDomainID=existing_domain,
+                AWSAccountId=env.AwsAccountId,
+                region=env.region,
+                RoleArn=env.EnvironmentDefaultIAMRoleArn,
+                owner=context().username,
+                SamlAdminGroupName=admin_group,
+                tags=request.tags,
+            )
+            SageMakerStudioRepository(session).save_sm_user(user=sagemaker_studio_user)
+
+            ResourcePolicy.attach_resource_policy(
+                session=session,
+                group=request.SamlAdminGroupName,
+                permissions=SGMSTUDIO_USER_ALL,
+                resource_uri=sagemaker_studio_user.sagemakerStudioUserProfileUri,
+                resource_type=models.SagemakerStudioUserProfile.__name__,
+            )
+
+            if env.SamlGroupName != sagemaker_studio_user.SamlAdminGroupName:
+                ResourcePolicy.attach_resource_policy(
+                    session=session,
+                    group=env.SamlGroupName,
+                    permissions=SGMSTUDIO_USER_ALL,
+                    resource_uri=sagemaker_studio_user.sagemakerStudioUserProfileUri,
+                    resource_type=models.SagemakerStudioUserProfile.__name__,
+                )
+
+            Stack.create_stack(
+                session=session,
+                environment_uri=sagemaker_studio_user.environmentUri,
+                target_type='sagemakerstudiouserprofile',
+                target_uri=sagemaker_studio_user.sagemakerStudioUserProfileUri,
+                target_label=sagemaker_studio_user.label,
+            )
+
+        stack_helper.deploy_stack(targetUri=sagemaker_studio_user.sagemakerStudioUserProfileUri)
+
+        return sagemaker_studio_user
+
+    @staticmethod
+    def list_sagemaker_studio_users(*, filter: dict) -> dict:
+        with _session() as session:
+            return SageMakerStudioRepository(session).paginated_sagemaker_studio_users(
+                username=context.username,
```

Comment:
I think it should be `context()` :)

Reply:
I am changing to use `get_context` directly to point out that it is a function